src/gui/guiSkin.cpp

```diff
@@ -0,0 +1,1065 @@
+// Copyright (C) 2002-2012 Nikolaus Gebhardt
+// Copyright (C) 2019 Irrlick

```

Comment:
Just to make sure: Is this line correct? (assuming `CGUISkin.cpp?  was directly copied over).

Reply:
Irrlick is a kidscode dev, the first commit is partially backported from kidscode

CGUISkin is hidden behind an interface, and cannot be derived from. Fuck Irrlicht